src/routes/(app)/login/+page.svelte

```diff
@@ -18,14 +20,37 @@
   }
 </script>
 
-<form on:submit={submitLogin}>
-  <input type="text" name="email" placeholder="email" bind:value={email} />
-  <br />
-  <input type="password" name="password" placeholder="password" bind:value={password} />
-  <br />
-  <button type="submit">Login</button>
-  <br />
-  <p>{message}</p>
-  <br />
-  <button><a href="/profile">Profile</a></button>
-</form>
+<!-- Colocar a página toda scrollable incluindo footer, ver container queries -->
```

Comment:
```suggestion
```

Reply:
is this still a problem? I don't think so

Reply:
It seems like it is a problem on really short screens, but a general one at that because it happens not only on this page.
![image](https://github.com/user-attachments/assets/eaa46226-8575-4099-b47b-5d74ebace2ce)
Do you think this is worth concerning about? I might open a new issue on it

Reply:
Ah I see! Open an issue, we can think about a fix later on